target_resci/__init__.py

```diff
@@ -323,7 +319,7 @@ def consume(self, reader):
         """Consume all the lines from the queue, flushing when done."""
         for line in reader:
             self.handle_line(line)
-        self.flush()
+        self.flush(True)
```

Comment:
Normally, `flush()` will just write records to a file. We set `final_batch` to `True` when we're done reading all lines from stdin, which triggers the api call.